src/MCPClient/lib/clientScripts/moveToBacklog.py

```diff
@@ -6,11 +6,13 @@
 import sys
 
 # archivematicaCommon
+from custom_handlers import GroupWriteRotatingFileHandler
 import storageService as storage_service
 
-logger = logging.getLogger(__name__)
-logging.basicConfig(filename="/tmp/archivematica.log",
-    level=logging.INFO)
+logger = logging.getLogger('archivematica.mcp.client')
+logger.addHandler(GroupWriteRotatingFileHandler("/var/log/archivematica/dashboard/dashboard.log",
```

Comment:
I think I'd rather ```/var/log/archivematica/MCPClient/clientscripts.log``` and have the logger be based on the filename.

Well, really I'd rather have this configured once in the ```__init__.py``` and everything just does ```LOGGER = logging.getLogger(__file__)```, but baby steps.